Gruntfile.coffee

```diff
@@ -54,6 +54,10 @@ module.exports = (grunt) ->
         src: 'dist/client.js'
         dest: 'dist/client.min.js'
 
+      instrumentation_jquery:
```

Comment:
No, it is adding minified version with sourcemaps - jquery.min.js and jquery.min.map